sympy/functions/elementary/piecewise.py

```diff
@@ -462,20 +465,22 @@ def _eval_subs(self, old, new):
     def _eval_transpose(self):
         return self.func(*[(e.transpose(), c) for e, c in self.args])
 
-    def _eval_template_is_attr(self, is_attr, when_multiple=None):
```

Comment:
Is it doing anything smart enough to check if `x.is_negative`, or is that outside the scope of this PR?